### PR TITLE
Add support for /deep/ selector in sass style files.

### PR DESCRIPTION
**Pull request by iFwu**
Since sass parser doesn't allow '>>>' selector, vue-loader uses /deep/ instead.
I just looked into vue-loader's code and implement it in this loader.

### CODE REVIEW

File: lib/modules/add-scoped-id.js [hunk 1]
```diff
@@ -3,12 +3,27 @@ const postcss = require('postcss')
 const selectorParser = require('postcss-selector-parser')
 
 function isDeepCombinator (node) {
-  return node.type === 'combinator' && node.toString().trim() === '>>>'
+  if (node.type === 'combinator' && node.toString().trim() === '>>>') {
+    return true
+  }
+  if (node.type === 'tag' && node.toString().trim() === '/deep/') {
```
Comment (ktsn) [inline lib/modules/add-scoped-id.js]: Can we reuse `isSassDeepCombinator` here?

File: lib/modules/add-scoped-id.js [hunk 2]
```diff
@@ -3,12 +3,27 @@ const postcss = require('postcss')
...
+    return true
+  }
+}
+
+function isSassDeepCombinator (node) {
```
Comment (ktsn) [inline lib/modules/add-scoped-id.js]: It would be better to choose other name because `/deep/` is not sass specific combinator.

File: lib/modules/add-scoped-id.js [hunk 3]
```diff
@@ -43,14 +58,13 @@ const addScopedIdPlugin = postcss.plugin('add-scoped-id', options => {
   const selectorTransformer = selectorParser(selectors => {
     selectors.each(selector => {
       const target = getTargetNode(selector)
-
+      replaceDeepCombinator(selector)
```
Comment (ktsn) [inline lib/modules/add-scoped-id.js]: Why did you move this here?

Comment (xekri) [reply lib/modules/add-scoped-id.js]: If it doesn't change the order, `/deep/` will be treated as a tag and be added with scoped id.

**Comment by ktsn** [comment]
Thanks for your contribution. vue-template-loader now uses [@vue/component-compiler-utils](https://github.com/vuejs/component-compiler-utils) which officially provides SFC specific features. So this loader has `/deep/` selector now.
